apps/style/common.scss

```diff
@@ -270,6 +271,7 @@ html[dir='rtl'] #codeWorkspace {
   .ace_scroller, { // droplet code view
     background-color: rgba(0, 0, 0, 0.1);
   }
+  // #design-mode-container manages this from within the react component
```

Comment:
My original approach was to try and have design-mode-container behave similarly to blocklySvg, droplet-main-scroller, ace-scroller via CSS. I ended up getting away from that, but still wanted to be sure that if we were to change this CSS we update the design-mode-container code as well. I'll try to make this more clear.